src/Sylius/Component/Registry/DependencyInjection/Compiler/AliasedServicePass.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+
+namespace Sylius\Component\Registry\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+use Sylius\Component\Registry\ServiceRegistryInterface;
+
+/**
+ * This compiler pass will add services to a registered ServiceRegistry
+ * definition.
+ *
+ * @author Daniel Leech <daniel@dantleech.com>
+ */
+class AliasedServicePass implements CompilerPassInterface
+{
+    private $serviceRegistryId;
+    private $serviceTag;
+    private $options;
+
+    /**
+     * @param string $serviceRegistryId
+     * @param string $serviceTag
+     * @param array $options
+     */
+    public function __construct(
+        $serviceRegistryId,
+        $serviceTag,
+        array $options = []
```

Comment:
Im would vote to add them to constructor with some default value. On the other hand, I'm not convinced about defining one of the options, just because it is declared before the option that I want to define...